src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { ChangeEventHandler, FormEventHandler, useState } from 'react';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+
+// eslint-disable-next-line max-len
+const DEFAULT_PREVIEW = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+interface Props {
+  onMovieAdd: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ onMovieAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const [field, setField] = useState('');
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const handleSubmit: FormEventHandler<HTMLFormElement> = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    getMovie(field)
+      .then((response: MovieData | ResponseError) => {
+        if ('Error' in response) {
```

Comment:
```suggestion
        if (response instanceof ResponseError) {
```

Reply:
ResponseError is interface, we can't use instanceOf